### ISSUE TITLE
can't launch binary

### ISSUE THREAD

**Issue by bobblkabb** [hyprwm/Hyprland#700]
Performed 'manual installation' (copy Hyperland, hyperctl, libwlroots.so.11032, .conf and .desktop) but can't launch hyprland on openSUSE Leap 15.4:
`Hyprland: error while loading shared libraries: libwlroots.so.11032: cannot open shared object file: No such file or directory`

**Comment by vaxerski** [comment]
well you didnt copy / copied to the wrong place `libwlroots.so.11032`, no?

**Comment by bobblkabb** [comment]
To /usr/lib like described.
_..copy the wlroots .so (wlroots.so.XX032) to /usr/lib/._

**Comment by vaxerski** [comment]
maybe try `/usr/local/lib/`

**Comment by bobblkabb** [comment]
Now I'm getting:
```
Hyprland: /lib64/libc.so.6: version `GLIBC_2.34' not found (required by Hyprland)
Hyprland: /lib64/libc.so.6: version `GLIBC_2.32' not found (required by Hyprland)
Hyprland: /lib64/libc.so.6: version `GLIBC_2.33' not found (required by Hyprland)
Hyprland: /lib64/libc.so.6: version `GLIBC_2.33' not found (required by /usr/lib/libwlroots.so.11032)
Hyprland: /lib64/libc.so.6: version `GLIBC_2.34' not found (required by /usr/lib/libwlroots.so.11032)
```
Hmm
openSUSE comes with glibc- 2.31.

**Comment by vaxerski** [comment]
unlucky then. Means suse's glibc is pretty ooold. 

iirc, the SUSE maintainer has mentioned it only works on tumbleweed.

**Comment by viperML** [comment]
No, that is why you don't build binaries against arch and claim that it works on any distro

**Comment by vaxerski** [comment]
ok mr big smarty, hyprland requires gcc from may this year. I am pretty sure Void has gcc and stdlib++ older than this, so it's not unreasonable to assume SUSE has too.

**Comment by viperML** [comment]
Your point is?

**Comment by bobblkabb** [comment]
So there's nothing I can do than wait. Anyway, thank you for the support. 
Hyprland is the hottest shit since sliced bread. Keep on the great work.

**Comment by vaxerski** [comment]
you can compile gcc 12.1 yourself, technically.

**Comment by bobblkabb** [comment]
Think I'll wait :-)

**Comment by TheComputerM** [comment]
I copied libwlroots.so.11032 to both /usr/lib and /usr/local/lib but I am still getting the same error of cannot open shared object file: No such file or directory.

I am on Fedora 37 btw.

**Comment by vaxerski** [comment]
cannot open *what* so file? Fedora has some outdated deps.

**Comment by TheComputerM** [comment]
I am getting this exact error: `Hyprland: error while loading shared libraries: libwlroots.so.11032: cannot open shared object file: No such file or directory`

**Comment by vaxerski** [comment]
make sure the so is in ldpath

**Comment by TheComputerM** [comment]
the `$LD_LIBRARY_PATH` was unset so I ran `sudo ldconfig` and then I got another error saying that `libseat.so.0` was missing but that was fixed by running `sudo dnf install libseat`. Now I have another error saying `libxcb-errors.so.0 is missing or not found`. I cannot find that package anywhere in the fedora repos or anything similar to it. I tried building from source but I could not build and am getting obscure errors when running automake.

**Comment by vaxerski** [comment]
see the fedora discussion.

**Comment by TheComputerM** [comment]
Yeah saw that but my dumbass forgot to run sudo before building. Now when running the Hyprland command getting this in red text: `[types/wlr_drm_lease_v1.c:713] No DRM backend supplied, failed to create wlr_drm_lease_v1_manager`. I think libdrm is outdated, or is there some way to fix this. Thanks for your help, learned a lot about linux.

**Comment by vaxerski** [comment]
no clue

**Comment by ancion** [comment]
> 

Did you complier libxcb-errors.so.0 file successfully? If you will share with me, that should be much thank for you, Because I don't want to build it from source !!
